### PR TITLE
[PYTHON] Changes input variable to not conflict with built-in function

### PR DESCRIPTION
**Pull request by DylanGuedes**
Signed-off-by: DylanGuedes <djmgguedes@gmail.com>

## What changes were proposed in this pull request?

Changes variable name conflict: [input is a built-in python function](https://stackoverflow.com/questions/20670732/is-input-a-keyword-in-python).

## How was this patch tested?

I runned the example and it works fine.

### CODE REVIEW

**Comment by HyukjinKwon** [comment]
ok to test

File: examples/src/main/python/ml/dataframe_example.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 """
 An example of how to use DataFrame for ML. Run with::
-    bin/spark-submit examples/src/main/python/ml/dataframe_example.py <input>
+    bin/spark-submit examples/src/main/python/ml/dataframe_example.py <dataset>
```
Comment (HyukjinKwon) [inline examples/src/main/python/ml/dataframe_example.py]: Shall we change `dataset` to like .. `input_path` or `path`? I think the name `dataset` is usually used for `Dataset` instance.

**Comment by HyukjinKwon** [comment]
I think it's fine. It should better avoid shadowing builtin names but can you check out other examples too while we are here?

**Comment by DylanGuedes** [comment]
Sure, and thanks for the review! I used the "dataset" name because I checked a few examples that also use it, but I'll rename to path then. I already checked all other examples using grep and this is the only one using "input" name.

**Comment by DylanGuedes** [comment]
@HyukjinKwon I checked every other built in function and this was the only one being used. What you think?

File: examples/src/main/python/ml/dataframe_example.py [hunk 2]
```diff
@@ -35,18 +35,18 @@
...
 
     spark = SparkSession \
         .builder \
         .appName("DataFrameExample") \
         .getOrCreate()
 
-    # Load input data
-    print("Loading LIBSVM file with UDT from " + input + ".")
-    df = spark.read.format("libsvm").load(input).cache()
+    # Load file from path
```
Comment (HyukjinKwon) [inline examples/src/main/python/ml/dataframe_example.py]: Let's just write it as `Load an input file`.

**Review by HyukjinKwon — APPROVED** [review]
LGTM

File: examples/src/main/python/ml/dataframe_example.py (hunk 2 above)
Comment (DylanGuedes) [reply examples/src/main/python/ml/dataframe_example.py]: Done.

**Review by felixcheung — APPROVED** [review]
can you update the PR title to say [PYTHON] example?

**Comment by DylanGuedes** [comment]
Sure. Done.

**Comment by HyukjinKwon** [comment]
Merged to master and branch-2.3.

**Pull request closed without merging**